components/Home.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Appbar, IconButton, Text, Subheading, Menu } from 'react-native-paper'
+import { useAuthenticationContext } from '../context/authentication'
```

Comment:
let's use the eslint import rule and configure the order so that external imports are separated from internal imports by a blank line